_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,304 @@
+
+        - It'll create confusion
+
+        - It'll use more block space
+
+        - It [reduces][poelstra anon] the size of taproot's anonymity
+          set, both directly and when protocols such as [ring signature
+          membership proofs][nick ring] or [Provisions][] are being used
+
+    - *Bandwidth/storage costs versus CPU costs:* it's [possible][rubin
+      recovery] to eliminate the extra 32-byte storage overhead from
+      hash indirection by deriving the public key from a signature and
+      the transaction data it signs, a technique called *key recovery*.
+      Again, this was opposed.  Key recovery requires a [significant amount of CPU][corallo
+      recovery overhead] that would slow down nodes and also prevents
+      the use of schnorr batch validation that can make historic block
+      verification up to three times faster.  It also [makes][poelstra
+      slowdowns] anonymous membership proofs and related techniques both
+      harder to develop and much more CPU intensive.  There
+      [may][poelstra patent story] also be a [patent][US7215773B1] issue.
+
+  As of this writing, it appears the mailing list discussion has
+  concluded without any obvious loss of community support for taproot.
+  As researchers and businesses continue improving the state of the art
+  in quantum computing, we expect to see future discussions about how to
+  best keep Bitcoin secure.
+
+- **Weekly taproot activation meetings:** ten weekly meetings to discuss
+  details related to activating [taproot][topic taproot] have been
+  scheduled for each Tuesday at 19:00 UTC in the
+  [##taproot-activation][] IRC channel, with the first
+  [meeting][activation meeting log] occurring yesterday (March 23rd).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20861][] implements support for BIP350 (Bech32m format for v1+
+  witness addresses). Bech32m supersedes bech32 as the address format for
+  native segwit outputs of version 1-16. Native segwit version 0 outputs,
+  P2WPKH and P2WSH, will continue to use bech32. In effect, this PR will enable
```

Comment:
Perhaps:

```suggestion
  native segwit outputs of version 1-16. Native segwit version 0 outputs
  (P2WPKH and P2WSH) will continue to use bech32. In effect, this PR will enable
```

otherwise "Native segwit version 0 outputs, P2WPKH and P2WSH" looks a bit like a list when parsing the sentence for the first time.

Reply:
Ah yes, that's much better. Done.